src/components/editMeasure/measureDetails/stewardAndDevelopers/StewardAndDevelopers.test.tsx

```diff
@@ -147,17 +170,40 @@ describe("Steward and Developers component", () => {
   });
 
   it("should disable dropdowns if the user is not the owner of the measure", async () => {
```

Comment:
Recommend renaming as we're verifying edit permissions.
```suggestion
  it("should disable dropdowns if the user does not have measure edit permissions", async () => {
```

Reply:
fixed